core/src/com/unciv/logic/automation/ConstructionAutomation.kt

```diff
@@ -187,6 +186,11 @@ class ConstructionAutomation(val cityConstructions: CityConstructions){
         if (preferredVictoryType == VictoryType.Cultural
                 && wonder.name in listOf("Sistine Chapel", "Eiffel Tower", "Cristo Redentor", "Neuschwanstein", "Sydney Opera House"))
             return 3f
+        // Only start building if we are the city that would complete it the soonest
+        if (wonder.uniques.contains("Triggers a Cultural Victory upon completion") && cityInfo == civInfo.cities.minByOrNull {
+                cityInfo.cityConstructions.turnsToConstruction(it.cityConstructions.getCurrentConstruction().name) + cityInfo.cityConstructions.turnsToConstruction(wonder.name) 
```

Comment:
Surely we don't care about the current construction, since we're attempting to replace it?

Reply:
Ah, I see what happened here, I messed up the last update, I'll fix it this night